packages/docs-app/src/examples/core-examples/buttonGroupPopoverExample.tsx

```diff
@@ -23,33 +21,28 @@ export interface IButtonGroupPopoverExampleState {
 export class ButtonGroupPopoverExample extends React.PureComponent<IExampleProps, IButtonGroupPopoverExampleState> {
     public state: IButtonGroupPopoverExampleState = {
         alignText: Alignment.CENTER,
-        intent: Intent.NONE,
         large: false,
         minimal: false,
         vertical: false,
     };
 
-    private handleIntentChange = handleStringChange((intent: Intent) => this.setState({ intent }));
     private handleLargeChange = handleBooleanChange(large => this.setState({ large }));
     private handleMinimalChange = handleBooleanChange(minimal => this.setState({ minimal }));
     private handleVerticalChange = handleBooleanChange(vertical => this.setState({ vertical }));
 
     public render() {
-        const { intent, ...bgProps } = this.state;
         const options = (
             <>
                 <H5>Props</H5>
                 <Switch label="Large" checked={this.state.large} onChange={this.handleLargeChange} />
                 <Switch label="Minimal" checked={this.state.minimal} onChange={this.handleMinimalChange} />
                 <Switch label="Vertical" checked={this.state.vertical} onChange={this.handleVerticalChange} />
                 <AlignmentSelect align={this.state.alignText} onChange={this.handleAlignChange} />
-                <H5>Button props</H5>
-                <IntentSelect intent={this.state.intent} onChange={this.handleIntentChange} />
```

Comment:
reducing vertical height of this example, and this feature is just not critical to demo.